src/igvfd/tests/data/inserts/assay_term.json

```diff
@@ -45,6 +45,10 @@
         "uuid": "0393b224-8f87-5e94-7aeb-faff044f707d",
         "aliases": [
             "igvf:CRISPR"
+        ],
+        "preferred_assay_titles": [
```

Comment:
Do these need to be `preferred_assay_title`?

Reply:
we are change preferred_assay_title -> preferred_assay_titles under measurement set